packages/ui/src/routes/profile/+page.svelte

```diff
@@ -24,16 +24,13 @@
 			error = err as Error
 		}
 	}
+
+	export let onBack: () => unknown = () => history.back()
```

Comment:
```suggestion
```

Reply:
While it is fine to define variables and functions that you later use, it does not usually make sense if they are this small and only used once or twice.

Also (and this is svelte thing) you should not use the `export` here. The world `export` only makes sense in components and means that you are defining the variable/function as a property that can be passed to the component. `+page.svelte` is not instantiated by us so we do not pass it any properties.

Reply:
Perfect! Got it.